logger/testk8s/kafka-broker-deployment.yaml

```diff
@@ -33,3 +33,5 @@ spec:
                 fieldPath: status.podIP
           - name: KAFKA_ZOOKEEPER_CONNECT
             value: zookeeper:2181
+          - name: KAFKA_CREATE_TOPICS
+            value: logs:3:1
```

Comment:
We run a proper cluster in prod so partitions and replication should be variables